cloudflare/schema_cloudflare_access_group.go

```diff
@@ -218,5 +218,21 @@ var AccessGroupOptionSchemaElement = &schema.Resource{
 				},
 			},
 		},
+		"external_evaluation": {
+			Type:     schema.TypeList,
+			Optional: true,
```

Comment:
just for good measure

```suggestion
			Optional: true,
			MaxItems: 1,
```